js/views/settingsVw.js

```diff
@@ -236,6 +239,22 @@ module.exports = pageVw.extend({
         placeholder_text_multiple: window.polyglot.t('chosenJS.placeHolderTextMultiple')
       });
 
+      var connectedPeers = "";
+      $.ajax({
+        url: self.serverUrl+ "routing_table",
+        success: function(data){
+          self.$el.find('span.js-numConnectedPeers').text(data.length);
```

Comment:
The `span` in the selector is redundant and, if we change the markup to a different element, it would require the JS code to change. Can we eliminate the span part? The class part is sufficient here.